src/game.py

```
@@ -29,8 +31,21 @@ def get_airports(self):
         for airport in db_airports:
-            self.airports.append(Airport(airport[0], airport[1], airport[2], airport[3], airport[4]))
+
+            # TODO: create a random resource list
+            random_resources = random.choices(inventory_items, k=2)
+            resources_with_values = []
+            for resource in random_resources:
+                max_random_value = 0
+                if resource == 'weapon':
+                    max_random_value = 5
+                else:
+                    max_random_value = 50
```

Ruff (0.8.2)

Use ternary operator `max_random_value = 5 if resource == "weapon" else 50` instead of `if`-`else`-block

Replace `if`-`else`-block with `max_random_value = 5 if resource == "weapon" else 50`

(SIM108)